src/Console/Command/HelpCommand.php

```diff
@@ -167,7 +167,7 @@ public static function getHelpCopy()
         ->in(__DIR__)
     ;
 
-    return PhpCsFixer\Config::create()
+    return (new PhpCsFixer\Config('foo'))
```

Comment:
I would remove `'foo'` from examples. Maybe add a dedicated example to show that a name can be passed but not really important IMO.

Reply:
Nah, I don't think it needs dedicated example, I removed that. :)

Reply:
Can you remove it from other examples too please?

Reply:
Sorry I'm a little ill today, fever's taking over ^^

Yay, done. :)